app/views.py

```diff
@@ -78,6 +78,80 @@ def admin():
     return render_template("admin.html", vehicles=vehicles, services=services)
 
 
+@app.route("/admin/services")
+def admin_services_management():
+    services = services_model.all()
+    return render_template("admin/services/index.html", services=services)
+
+
+@app.route("/admin/services/adding", methods=["GET", "POST"])
```

Comment:
hmmm... I prefer the name of the page to be `add` instead of `adding`

Reply:
I'll update it